src/app/recipient/components/recipient-earned-badge-list/recipient-earned-badge-list.component.html

```diff
@@ -1,255 +1,220 @@
-						/>
-						<svg icon="icon_list"></svg>
-					</label>
 				</div>
-			</div>
+			</ng-template>
 
-			<!-- Grid View -->
-			<ng-template [ngIf]="badgesDisplay == 'grid'">
-				<!-- ------------------ UNGROUPED ------------------  -->
-				<ng-template [ngIf]="!groupByIssuer">
-					<div class="l-grid l-grid-large">
-						<!-- TODO: Fix the badgecard link for sharing and routing -->
-						<bg-badgecard
-							*ngFor="let badgeResult of badgeResults"
-							[badgeTitle]="badgeResult.badge.badgeClass.name"
-							[badgeImage]="badgeResult.badge.image"
-							[badgeDescription]="badgeResult.badge.badgeClass.description"
-							[badgeIssueDate]="badgeResult?.badge?.issueDate"
-							[mostRelevantStatus]="badgeResult.badge.mostRelevantStatus"
-							[issuerTitle]="badgeResult.badge.badgeClass.issuer.name"
-							[issuerSlug]="badgeResult.badge.badgeClass.issuer.slug"
-							[badgeSlug]="badgeResult.badge.slug"
-							(shareClicked)="shareBadge(badgeResult.badge)"
-						></bg-badgecard>
+			<div class="oeb-section-sm">
+				<oeb-backpack-tabs (onTabChanged)="onTabChange($event)" [tabs]="tabs"></oeb-backpack-tabs>
+				<!-- <ng-template #overViewTemplate>
+					<div class="oeb-section">
+						<h1 hlmH1 class="tw-text-purple">Coming soon...</h1>
 					</div>
-				</ng-template>
-				<!-- ------------------ GROUP BY ISSUER ------------------  -->
-				<ng-template [ngIf]="groupByIssuer">
-					<div *ngFor="let issuerGroup of issuerResults">
-						<h3 class="u-text-h3-semibold u-margin-bottom2x u-margin-top6x u-text-dark1">
-							{{ issuerGroup.issuer.name }}
-							<span class="u-text-small-semibold-caps u-text-dark4 u-margin-left2x"
-								>{{ issuerGroup.badges.length }}
-								{{ issuerGroup.badges.length == 1 ? 'Badge' : 'Badges' }}</span
-							>
-						</h3>
-						<div class="l-grid l-grid-large">
-							<bg-badgecard
-								*ngFor="let badge of issuerGroup.badges"
-								[badgeTitle]="badge.badgeClass.name"
-								[badgeImage]="badge.image"
-								[badgeDescription]="badge.badgeClass.description"
-								[badgeIssueDate]="badge.issueDate"
-								[mostRelevantStatus]="badge.mostRelevantStatus"
-								[issuerSlug]="issuerIdToSlug(issuerGroup.issuerId)"
-								[issuerTitle]="badge.badgeClass.issuer.name"
-								[badgeSlug]="badge.slug"
-								(shareClicked)="shareBadge(badge)"
-							></bg-badgecard>
+				</ng-template> -->
+				<ng-template #badgesTemplate>
+					<ng-template [bgAwaitPromises]="[badgesLoaded]">
+						<ng-container *ngIf="mozillaFeatureEnabled">
+							<div *ngIf="!mozillaTransitionOver" #mozzBanner>
+								<div class="notification notification-is-active notification-info">
+									<div class="notification-x-icon">
+										<svg icon="icon_info"></svg>
+									</div>
+									<div class="notification-x-text">
+										<h2>{{ 'RecBadge.fromMozilla' | translate }}</h2>
+										<p>
+											{{ 'RecBadge.finisihAddBadge1' | translate }}
```

Comment:
Typo

Reply:
(Also for 2 and 3)

Reply:
This was some old code. I just removed it